contracts/stake-cw20-external-rewards/src/contract.rs

```diff
@@ -275,7 +286,9 @@ pub fn get_rewards_earned(
     let user_reward_per_token = USER_REWARD_PER_TOKEN
         .load(deps.storage, addr.clone())
         .unwrap_or_default();
-    let reward_factor = reward_per_token.checked_sub(user_reward_per_token)?.checked_div(scale_factor())?;
+    let reward_factor = reward_per_token
+        .checked_sub(user_reward_per_token)?
+        .checked_div(scale_factor())?;
```

Comment:
fwiw, the regular divide / multiply / add implementations for Uint128 also use checked operations. Might make the code easier to read to just use those though you trade a returned error for a panic. 🤷‍♂️